test/blackbox-tests/test-cases/describe/describe-pp/describe-pp.t/run.t

```diff
@@ -7,3 +7,12 @@ We can also show the original source if it is not preprocessed
 
   $ dune describe pp src/util.ml
   let log str = print_endline str
+
+We also make sure that the dump file is not present
+
+  $ ls
```

Comment:
```suggestion
  $ dune_cmd exists profile.dump
```

I'm not sure the output of `ls` is stable enough for this

Reply:
done.